pydrive/files.py

```diff
@@ -64,9 +64,11 @@ def _GetList(self):
     self['corpus'] = 'DEFAULT'
     self['supportsTeamDrives'] = True
     self['includeTeamDriveItems'] = True
-
-    self.metadata = self.auth.service.files().list(**dict(self)).execute(
-      http=self.http)
+    try:
```

Comment:
@MaxRis let's merge this into PyDrive2 and release a new version?

Reply:
@MaxRis https://github.com/iterative/PyDrive2/pull/5 please take a look at this one.

Reply:
@shcheklei cool, thanks for merging!